src/main/java/org/tron/common/runtime/vm/VMConstant.java

```diff
@@ -0,0 +1,8 @@
+package org.tron.common.runtime.vm;
+
+public class VMConstant {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1633&issues=AWalyDXvdPPRVdwylpM2&open=AWalyDXvdPPRVdwylpM2)

[ISSUE_KEY]: AWalyDXvdPPRVdwylpM2